### ISSUE TITLE
screenReader JAWS reads out content twice in FireFox with enterMode CKEDITOR.ENTER_BR

### ISSUE THREAD

**Issue by JulesAchiel** [ckeditor/ckeditor-dev#938]
## Are you reporting a feature request or a bug?
bug

## [Check if the issue is already reported](http://docs.ckeditor.com/#!/guide/dev_issues_readme-section-avoid-duplicates)
none found

## Provide detailed reproduction steps (if any)

1. Open [CKEditor in BR mode](https://codepen.io/mlewand/pen/jzMbaa).
1. Start screenreader JAWS
1. put JAWS cursor focus on the header
1. use the down arrow keys to move to the next object for JAWS to read out; keep pressing through the complete toolbar
1. When the content is read out you will notice it is done twice
1. When taking away the configuration `CKEDITOR.ENTER_BR` (or use `ENTER_DIV`) this is only read out once.

### Expected result
JAWS to read out the CKEditor content only once

### Actual result
JAWS reads the CKEditor content twice

## Other details

* Browser: FireFox 52.3.0
* OS: N/A
* CKEditor version: 4.7.2
* Installed CKEditor plugins: none

**Comment by jswiderski** [comment]
When going through elements using down arrow, after you pass the toolbar, you will get:

**In BR mode:**
"lorem ipsum edit" (you press down arrow) "lorem ipsum edit"

**In Div mode:**
"edit" (you press down arrow) "lorem ipsum edit"

As you can see "lorem ipsum" is being read twice in BR mode

Plese see attached video (sorry for the voice quality): [modes.zip](https://github.com/ckeditor/ckeditor-dev/files/1318321/modes.zip)

**Comment by JulesAchiel** [comment]
Do we have an ETA of when this issue will be fixed?

**Comment by JulesAchiel** [comment]
When is this expected to be taken up for a fix?

**Comment by mlewand** [comment]
After deeper investigation, we're not using any attribute trickery here, yet JAWS adds "edit" parts.

It's basically a `div[role=application]` containing an `iframe`. Inside of application there's a body with `contenteditable` on it, like that:

```
<body contenteditable="true" class="cke_editable cke_editable_themed cke_contents_ltr" spellcheck="false">…</body>
```

The "lorem ipsum" text sits inside, and is not duplicated anywhere. Yet JAWS is adding extra "lorem ipsum edit" fields.

I also used content with multiple lines (separated with `<br>` and it added multiple `<line> edit` members. JAWS should not add it here, and should read the content that actually is in markup without creating artificial/bogus elements. That's an upstream issue that should be fixed in JAWS directly.

I checked that NVDA does not have this issue, although NVDA simply skips the application frame when navigating in this way.
